static/main.css

```diff
@@ -57,10 +57,15 @@ ul {
   list-style: none;
   padding-left: 0px;
 }
-ul li button {
+ul li {
   min-width: 200px;
 }
 
+button {
+  /*  This broke emoji mart after update */
+  /* min-width: 200px; */
```

Comment:
For what is this used?

Reply:
I think I adjusted the width of the emoji mart component with this (maybe). I would keep it as it is, but try out to resize the window and see if everything still looks fine.